src/lambda/etherscan.ts

```diff
@@ -1,6 +1,8 @@
-const axios = require("axios")
+import axios from "axios"
 
-const lambda = async (apiKey) => {
+import type { HandlerResponse } from "@netlify/functions"
+
+const lambda = async (apiKey: string | undefined): Promise<HandlerResponse> => {
```

Comment:
`apiKey` should not be undefined. 
```suggestion
const lambda = async (apiKey: string): Promise<HandlerResponse> => {
```

And then in the handler function please do something like:
```ts
const handler = () => {
  let apiKey = process.env.ETHERSCAN_API_KEY

  if(!apiKey) {
    throw new Error('required env ETHERSCAN_API_KEY not set')
  }

  return lambda(apiKey)
}
```

Reply:
I also went ahead and added `environment.d.ts` (https://github.com/ethereum/ethereum-org-website/pull/6500/commits/571c2e5e3fab687c1b803b33f9cf0fad6cca5272)